netbox/templates/dcim/device_component_list.html

```diff
@@ -0,0 +1,19 @@
+{% extends '_base.html' %}
+{% load buttons %}
+
+{% block content %}
+<div class="pull-right noprint">
+    {% export_button content_type %}
+</div>
+<h1>{% block title %}{{ table.component_type }}{% endblock %}</h1>
```

Comment:
We should be able to reference the table's model here instead of setting `component_type` on each table. (You can use the `|model_name` filter from the `helpers` library to access the model's verbose name.)

Reply:
This is exactly what I was looking for.  ^_^